CONTRIBUTING.md

```diff
@@ -12,11 +12,25 @@ For other place, both English and Japanese are acceptable.
 
 ## issues / issue について
 
+Not only sending Pull Requests, feature requests and bug reports are welcome.
+
 機能要求やバグ報告は気軽にしてください。
 コードを書くことだけが開発ではありません。
 
 ## pull requests / プルリクについて
 
+PR is always welcome.
+
+However, please note that PR is not always merged as it is.
+To improve PR quality, reviewers may ask you change requests.
+
+-   Test your PR branch on local by `python3 setup.py test`.
+-   Write friendly code to the reviewers.
+    -   Diffs need to be limited to the PR purpose.   
+    -   Split commits appropriately.
+    -   Comment on the code where you are not confident of.
+-   If you want to add feature, it is desirable discuss in advance that before writing code.
```

Comment:
```suggestion
-   If you want to add a feature, it would be better to discuss before writing code
    -   because your feature is not always merged
```

-  `feature` は可算ぽい https://eow.alc.co.jp/search?q=feature
-   `in advance` と `before` は被ってる